addons/cppcheckdata.py

```diff
@@ -222,8 +222,8 @@ class Token:
         isSigned           Is this token a signed type
         isExpandedMacro    Is this token a expanded macro token
         isRemovedVoidParameter  Has void parameter been removed?
-        isSplittedVarDeclComma  Is this a comma changed to semicolon in a splitted variable declaration ('int a,b;' => 'int a; int b;')
-        isSplittedVarDeclEq     Is this a '=' changed to semicolon in a splitted variable declaration ('int a=5;' => 'int a; a=5;')
+        isSplittedVarDeclComma  Is this a comma changed to semicolon in a split variable declaration ('int a,b;' => 'int a; int b;')
```

Comment:
Maybe we should change the names of the flags as well.

Reply:
Have knowledge of C, but not C++. Do some TODO processors need to be added somewhere?

Reply:
I just noticed that this is `cppcheckdata.py` and not a `.cpp` file. This just mirrors some code in `token.h` (which maybe should be changed at some point). Feel free to ignore my comment.

Reply:
For completeness: The flags in `token.h` are `fIsSplitVarDeclComma` and `fIsSplitVarDeclEq` (so not in sync with `cppcheckdata.py`, should be adapted), while the accessor functions are still called `isSplittedVarDeclComma()` and `isSplittedVarDeclEq()`, ugh...